src/06-time-tracking/time-tracking.apib

```diff
@@ -176,6 +176,23 @@ Delete tracked time.
 
 + Response 204
 
+### timeTracking.resume [POST /timers.resume]
```

Comment:
B: should be `POST /timeTracking.resume` I think?

Reply:
true that